DXMainClient/DXGUI/Multiplayer/GameLoadingLobbyBase.cs

```diff
@@ -49,18 +49,18 @@ public GameLoadingLobbyBase(WindowManager windowManager, DiscordHandler discordH
         protected EnhancedSoundEffect sndLeaveSound;
         protected EnhancedSoundEffect sndMessageSound;
 
-        private XNALabel lblDescription;
-        private XNAPanel panelPlayers;
-        private XNALabel[] lblPlayerNames;
+        protected XNALabel lblDescription;
+        protected XNAPanel panelPlayers;
+        protected XNALabel[] lblPlayerNames;
```

Comment:
Is there a reason why these are protected? As far as I can see, they aren't referenced in derived classes.

Reply:
It is a change that would be needed to implement pings for game loading lobbies (if done in a way of appending ping number to player label), because it's not possible to reference them in child classes. Overall later commits have some changes which are not needed for current feature set (only regular lobbies get pings), but would be needed for further addition of pings in LAN lobbies and game loading lobbies (like LAN player ping update function). I guess it's safe to omit them but I don't see those hurt much so I left them in.